lib/trino-orc/src/main/java/io/trino/orc/reader/SliceDictionaryColumnReader.java

```diff
@@ -365,4 +367,10 @@ public long getRetainedSizeInBytes()
     {
         return INSTANCE_SIZE;
     }
+
+    private void updateMemory()
```

Comment:
`updateMemory` -> `updateMemoryContext`

Reply:
No need. The purpose of the method is to update the memory usage, so the name seems appropriate

Reply:
`updateMemoryUsage` may be more suitable, thanks for remind.